prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -1473,15 +1472,17 @@ int efa_rdm_ope_post_remote_write(struct efa_rdm_ope *ope)
 		if (OFI_UNLIKELY(!pkt_entry))
 			return -FI_EAGAIN;
 
+		/* Use bounce buffer (registered) for buffer/desc, user input doesn't matter */
```

Comment:
Nit:
> Use bounce buffer (registered) for buffer/desc, the buffer we pass to rdma-core doesn't matter

sounds better

Reply:
didn't quite use your words exactly, but made this more clear.